rules/falco_rules.yaml

```diff
@@ -3420,3 +3420,35 @@
 # Application rules have moved to application_rules.yaml. Please look
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
+
+
+- macro: rocketmq_check_netty_server
+  condition: (proc.name contains NettyServer and proc.env contains ROCKETMQ)
+
+- macro: rocketmq_check_payload_for_conf_update
+  condition: ((fd.typechar='4' or fd.typechar='6') and evt.type=read and evt.buffer contains 'code":25' and evt.buffer contains serializeTypeCurrentRPC and evt.buffer contains JSON)
```

Comment:
Maybe even worth a new macro `read_socket` or similar for, what is evt.dir here? 

```
evt.type=read and (fd.typechar='4' or fd.typechar='6') ...
```

Reply:
evt.dir is '<' exit events

Reply:
evt.dir is '<' exit events